src/poll.rs

```diff
@@ -17,11 +18,13 @@ use std::thread;
 use std::time::{Duration, Instant};
 use walkdir::WalkDir;
 
+#[derive(Debug)]
```

Comment:
This one isn't exposed, so Debug isn't necessary.

Reply:
This `Debug` impl is used by the `WatchData` impl